package.json

```diff
@@ -50,13 +29,39 @@
     "language",
     "Google Cloud Natural Language API"
   ],
+  "contributors": [
+    "Ace Nassri <anassri@google.com>",
+    "Amy <amy@infosleuth.net>",
+    "Dave Gramlich <callmehiphop@gmail.com>",
+    "Eric Uldall <ericuldall@gmail.com>",
+    "Ernest Landrito <landrito@google.com>",
+    "Gus Class <gguuss@gmail.com>",
+    "Jason Dobry <jason.dobry@gmail.com>",
+    "Jerjou <jerjou@users.noreply.github.com>",
+    "Jun Mukai <jun.mukai@gmail.com>",
+    "Luke Sneeringer <lukesneeringer@google.com>",
+    "Ricky Dunlop <hello@rickydunlop.co.uk>",
+    "Sawyer Hollenshead <sawyerh@users.noreply.github.com>",
+    "Song Wang <songwang@google.com>",
+    "Stephen Sawchuk <sawchuk@gmail.com>",
+    "Tim Swast <swast@google.com>"
+  ],
+  "scripts": {
```

Comment:
This part is a smidge concerning. Some repositories _will_ need one off scripts. This one, for instance, now has `slackbot-test`. It is going to be a problem if repo-tools plows over the entire `package.json`.

Reply:
repo-tools will not remove anything from package.json (in particular, does not take away or add anything to `scripts`), it will just add missing fields (like `engines`, `license`, etc.) and enforce a consistent ordering: important npm package metdata at the top, random extra crap in the middle, and scripts/deps at the bottom.

And I'm not seeing anything about `slackbot-test` in this `package.json` file.